vendor/github.com/integr8ly/grafana-operator/deploy/role.yaml

```diff
@@ -0,0 +1 @@
+roles/role.yaml
```

Comment:
I'm surprised at this file showing up here, I thought it would only be go files from consumed packages :thinking: